app/assets/javascripts/components/shapes/story.js

```diff
@@ -1,46 +1,29 @@
-import PropTypes from 'prop-types';
-import NotePropTypes from './note';
-import AttachmentPropTypes from './attachment';
-import TaskPropType from './task';
+import PropTypes from "prop-types";
```

Comment:
I noticed most of the changes in this file, just like in other JS files, are to use double quotes instead of single quotes. I think the linter is not configured correctly in your dev environment, and linter changes make it really hard to review. I am not sure if you will be able to roll back all the linter changes but maybe try not to include linter changes, please. You can have a separate PR to improve the linter or something.